docs/topics/templates.txt

```diff
@@ -611,6 +611,49 @@ creating an object that specifies the following attributes:
 
 .. currentmodule:: django.template
 
+.. _overriding-templates:
+
+Overriding templates
+====================
+
+To override templates from a package in ``django.contrib`` or from a third-party
+package, the preferred method is to create replacement templates in your
+project's templates directory.
+
+Let's say you're trying to override the templates for a third-party package
+called ``blog``, which provides the templates ``post.html`` and ``list.html``.
```

Comment:
For clarity, I think we should always refer to these templates as ``blog/post.html`` and ``blog/list.html``, never ``post.html`` and ``list.html``. It's clear from the directory structure you show later that that's the actual path to these templates. Since Django does _not_ do any automatic namespacing of template paths for templates within apps, and it's a common confusion point for newcomers who assume that it does, we should be careful not to encourage that mis-apprehension by giving unqualified paths to the templates.